spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -218,6 +254,7 @@ private void configureArmeriaService(ServerBuilder sb, ArmeriaSettings settings)
         configureServerWithArmeriaSettings(sb, settings,
                                            findBean(MeterRegistry.class).orElse(Metrics.globalRegistry),
                                            findBeans(HealthChecker.class));
+        configureTls(sb, settings.getSsl());
```

Comment:
Don't we need to call this only if `setting.getSsl()` returns non-null?

Reply:
Right, added null check